docker-compose.yml

```diff
@@ -12,10 +12,12 @@ mongodb:
   image: mongo:latest
   volumes_from:
     - mongodb-data
+  volumes:
+    - ./data/archive:/archive
```

Comment:
Yes, entirely. I use it to store a mongodump archive and to restore on the occasion I clear the volume (or want to restart from my snapshot). This isn't necessary, but I would recommend not exposing the db data files to the external host since Mongo uses memory mapped files and that isn't compatible in virtualized environments.